canonical-web-and-design/snapcraft.io
# Upload and edit banners

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1689
Fixes https://github.com/canonical-websites/snapcraft.io/issues/1690

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/<snap_name>/listing
- Follow the guidance displayed below the banner area "Show banner restrictions"
- Any file that doesn't meet the restrictions should fail with an error
- You should be able to set, remove and change the image
